frontend/src/components/Transactions/TransactionLink.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <div class="transaction-slot-link">
+    <div @click="visible = !visible">
+      <!-- Collaps Icon  -->
+      <collapse-icon class="text-right" :visible="visible" />
+      <div>
+        <b-row>
+          <!-- ICON  -->
+          <b-col cols="1">
+            <type-icon color="text-danger" icon="share" />
+          </b-col>
+
+          <b-col cols="11">
+            <!-- Amount / Name || Text -->
+            <amount-and-name-row :amount="amount" :text="$t('gdd_per_link.links_sum')" />
```

Comment:
the amount is not negative - clarify with @Mogge 

![image](https://user-images.githubusercontent.com/1238238/158023916-357b514b-0834-403f-b999-f85a7f41b596.png)

Reply:
You want a negative value to be returned? We should also indicate, that there is included the max decay for each pending transaction link.